README.md

```diff
@@ -32,16 +31,22 @@ Time to set up the server that will run Cartography.  Cartography _should_ work
 		2. Set up AWS credentials to this identity on your server, using a `config` and 	`credential` file.  For details, see AWS' [official guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
   
  	- **If you want to pull from multiple AWS accounts**, see [here](#multiple-aws-account-setup).
-
-
-3. If you're a GCP user, **prepare your GCP credential(s)**
+ 	
+ 3. If you're a GCP user, **prepare your GCP credential(s)**
 
     1. Create an identity - either a User Account or a Service Account - for Cartography to run as
     2. Ensure that this identity has the [securityReviewer](https://cloud.google.com/iam/docs/understanding-roles) role attached to it.
     3. Ensure that the machine you are running Cartography on can authenticate to this identity.  
         - **Method 1**: You can do this by setting your `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to a json file containing your credentials.  As per SecurityCommonSense™️, please ensure that only the user account that runs Cartography has read-access to this sensitive file.
         - **Method 2**: If you are running Cartography on a GCE instance or other GCP service, you can make use of the credential management provided by the default service accounts on these services.  See the [official docs](https://cloud.google.com/docs/authentication/production) on Application Default Credentials for more details.
 
+4. If you're a CRXcavator user, **prepare your CRXcavator API key**
+
+    1. Generate an API key from your CRXcavator [user page](https://crxcavator.io/user/settings#)
+    2. Populate the following environment variables in the shell running the sync service
+        1. CRXcavator_API_KEY - the full URL to the CRXcavator API. https://api.crxcavator.io/v1 as of 07/09/19
```

Comment:
I think this is a typo.  Did you mean `CRXCAVATOR_URL`?

Reply:
fixed